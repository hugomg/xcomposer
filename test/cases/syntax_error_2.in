U0041 A a\