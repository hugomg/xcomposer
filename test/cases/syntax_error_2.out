xcomposer encountered an error:

Syntax error in Line 1: backslash at end of string
